src/GraphQL.Relay/GraphQL.Relay.csproj

```diff
@@ -1,11 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net5</TargetFramework>
```

Comment:
Any reason we cannot target `netstandard2.0` ?  And why the package reference to `System.Linq.Queryable`?